src/ui/Base/ComponentOptions.ts

```diff
@@ -391,6 +561,24 @@ export class ComponentOptions {
     return ComponentOptions.buildOption<HTMLElement>(ComponentOptionsType.CHILD_HTML_ELEMENT, ComponentOptions.loadChildHtmlElementOption, optionArgs);
   }
 
+  /**
+   * Builds a template option.
+   *
+   * The option accepts a CSS selector matching a valid template. This selector can either be a class, or an ID
+   * selector.
+   *
```

Comment:
Can you please make sure the following information (about the Template Cache) is correct?

Thanks!